pages/Crewmates.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Head from 'next/head';
+
+// Components
+import Article from '../components/article';
+
+// Images
+const Founder = '/images/1.jpg';
+
+const CrewMates = () => (
+    <div className='  bg-dark-primary text-white'>
+        <Head>
+            <title>CrewMates </title>
+
+        </Head>
+
+        <div text-white text-6xl font-bold leading-none mb-6>CrewMates</div>
```

Comment:
Have you added the logo for crewmates here? And, what about the sub-heading. I can't find it.